src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortToPgen.java

```diff
@@ -212,7 +224,7 @@ protected void apply(VariantContext variantContext) {
     @Override
     public Object onTraversalSuccess() {
         super.onTraversalSuccess();
-        if (pgenWriter.getDroppedVariantCount() > 0l) {
```

Comment:
was this causing an existing bug, or just an aesthetic cleanup?

Reply:
Just an aesthetic cleanup. Suggested by IntelliJ.